rust/core-lib/src/file.rs

```diff
@@ -226,12 +226,9 @@ fn try_save(
 
     fs::rename(tmp_path, path)?;
 
-    if let Some(info_unwrapped) = file_info {
+    if let Some(_info) = file_info {
```

Comment:
That seems kinda hacky, it'd be best to only do the entire if clause on Unix.

Reply:
Agree, the solution should be to use the correct `#cfg[]` marker to skip the unused code on windows.

Reply:
@Cogitri @cmyr but `#cfg[]` tags don't work on if statements?

Reply:
@bzm3r than we can wrap the whole `if` in a block?

Reply:
@cmyr 🤦‍♂️ i'll do that!

Reply:
Yeah, this isn't right - the underscore prefix should only be used when the variable is actually not used, which is clearly not the case here. Probably easier is just to set `#[allow(unused)]`.